src/ui/popup/settings-page/invert-pdf.tsx

```diff
@@ -0,0 +1,21 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import CheckButton from '../check-button';
+
+export default function InvertPDF(props: ViewProps) {
+
+    function onInvertPDFChange(checked: boolean) {
+        props.actions.changeSettings({invertPDF: !checked});
+    }
+
+    return (
+        <CheckButton
+            checked={!props.data.settings.invertPDF}
+            label="Inverting PDF"
+            description={props.data.settings.invertPDF ?
+                "Disabled inverting PDF's" :
+                "Enabled inverting PDF's" }
```

Comment:
Maybe `Enabled inverting PDF files` is better for description. Also, some people (like me) prefer Light mode for PDFs, maybe the phrase should be `Enabled for PDF files`.